packages/pages-shared/metadata-generator/validateURL.ts

```diff
@@ -55,3 +55,8 @@ export const validateUrl = (
 			: 'URLs should either be relative (e.g. begin with a forward-slash), or use HTTPS (e.g. begin with "https://").',
 	];
 };
+
+export function urlHasHost(token: string): boolean {
+	const host = URL_REGEX.exec(token);
+	return Boolean(host && host.groups && host.groups.host);
```

Comment:
Why we wrapping this in another bool?
```suggestion
	return host && host.groups && host.groups.host;
```

Reply:
The return type without surrounding this in a Boolean is `boolean | null | undefined`, resulting ina type error

Reply:
Love TS